utils/build/docker/ruby/rails70.Dockerfile

```diff
@@ -10,6 +10,7 @@ COPY utils/build/docker/ruby/shared/rails/ .
 COPY utils/build/docker/ruby/install_ddtrace.sh binaries* /binaries/
 RUN /binaries/install_ddtrace.sh
 
+ENV RAILS_LOG_TO_STDOUT=true
```

Comment:
This set the value for all scenario, could you add it at scenario level. Ping me if you need spme help to achieve that